addons/gcp/external-dns/variables.tf

```diff
@@ -0,0 +1,73 @@
+variable "cloud" {
+  default = null
+}
+
+variable "cluster_id" {
+  default = null
+}
+
+variable "name_prefix" {
+  description = "name prefix"
+}
+
+variable "output_path" {
+  default = "../../../outputs"
+}
+
+variable "tsb_image_sync_username" {
+}
+
+variable "gcp_k8s_regions" {
+  default = []
+}
+
+locals {
+  k8s_regions = var.gcp_k8s_regions
+}
+
+variable "tetrate_owner" {
+    default = null
+}
+variable "tetrate_team" {
+    default = "tetrate-service-bridge-sandbox"
+}
```

Comment:
As with other providers, remove the defaults

Reply:
agree, resolved.